src/ast/ast.c

```diff
@@ -490,7 +490,7 @@ inline AST_AnnotationCtxCollection *AST_GetAnnotationCtxCollection(AST *ast) {
 
 static inline char *_create_anon_alias(int anon_count) {
 	char *alias;
-	asprintf(&alias, "anon_%d", anon_count);
+	asprintf(&alias, "@anon_%d", anon_count);
```

Comment:
So now all of our anonymous aliases are of the form `@anon_<id>` ?
I'll assume the user can't create an alias which starts with `@` ?

Reply:
yes the user cannot create alias like this